docs/docs/setting-up-ci-cd.mdx

```diff
@@ -36,116 +37,73 @@ resource-intensive tests only when certain files have been changed or when some
 other indicator is present. For example, if your code is hosted on Github,
 you can make a test run only if the pull request has a certain label (e.g. “NLU testing required”).
 
-### Validating Data and Stories
-
-Data validation verifies that no mistakes or major inconsistencies appear in your domain, NLU 
-data, or story data. To validate your data, have your CI run this command:
-
-```bash
-rasa data validate
+### CI Pipeline Overview
+
+There are different approaches to applying a CI pipeline depending on the number of changes.
+All follow a mix of training the new model, testing to compare to older versions,
+and setting up an easy CD pipeline to upload the new model.
+
+The first step after saving new training data is to kick off the pipeline. (Is it usually kicked off
+manually or triggered by an event like saving a new file?)
+(need different pipelines / options based on how big the changes are - also should we lay out how
+small or big of changes warrant which tests?) Typically this is done by xyz
+
+Next, you need to pick what level of tests to run based on the amount of data changed. The various types
+are laid out in [Testing Your Assistant](https://rasa.com/docs/rasa/testing-your-assistant), but a good
+rule of thumb is to always run your test stories to ensure no bugs were introduced. Larger changes
+may need a full NLU test with cross validation to look out for confusion between intents.
+
+After testing and posting your results for your team to see, have a person or team of reviewers
+to check the test results with your developer. An easy process to help this is to have your CI
+pipeline automatically create a pull request with the test results in a comment. That way, the
+reviewer can see the changes along with the improved results.
+
+Once the reviewer merges you can then initiate your CD pipeline to add the new model
+to another environment.
+
+### GitHub Actions CI Pipeline
+
+You can use the [Rasa Train-Test Github Action](https://github.com/RasaHQ/rasa-train-test-gha)
+in your CI pipeline to automatically perform data validation, training, and testing.
+
+An example CI pipeline using the Github Action is shown below:
+
+```yaml
+jobs:
+  training-testing:
+    name: Training and Testing
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Rasa Train and Test GitHub Action
+        uses: RasaHQ/rasa-train-test-gha@main
+        with:
+          requirements_file: requirements.txt
+          test_type: all
```

Comment:
I think we should add `data_validate: true` and `rasa_train:true` before this line. Even though though the default is true.

Reply:
@b-quachtran can you take this one?

Reply:
Yep ✅